apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/group/GroupChatInfoView.kt

```diff
@@ -394,7 +417,18 @@ fun ModalData.GroupChatInfoLayout(
       }
       val footerId = if (groupInfo.businessChat == null) MR.strings.only_group_owners_can_change_prefs else MR.strings.only_chat_owners_can_change_prefs
       SectionTextFooter(stringResource(footerId))
-      SectionDividerSpaced(maxTopPadding = true)
+      SectionDividerSpaced(maxTopPadding = true, maxBottomPadding = false)
+
+      SectionView {
+        TtlOptions(
+          chatItemTTL,
+          enabled = remember { mutableStateOf(true) },
+          onSelected = setChatItemTTL,
+          default = chatModel.chatItemTTL
+        )
+      }
+      SectionTextFooter(stringResource(MR.strings.chat_ttl_options_footer))
+      SectionDividerSpaced(maxTopPadding = true, maxBottomPadding = false)
```

Comment:
there is a smaller padding on bottom than on top. Probably, `maxBottomPadding` should be `true`?
On direct chat the padding looks ok